.coderabbit.yaml

```diff
@@ -0,0 +1,224 @@
+
+        Consider these 'requirements.txt' files the records of truth regarding project dependencies.
+    - path: tests/requirements.txt
+      instructions: >-
+        * The multicast project's own dependencies are recorded in
+        './requirements.txt' for production code.
+
+        * The multicast project's own test code dependencies are recorded in
+        'tests/requirements.txt' for testing the project.
+
+        Consider these the records of truth regarding project test code dependencies.
+    - path: .github/**
+      instructions: >-
+        * When the project is hosted on GitHub: All GitHub-specific configurations, templates, and tools should be found in the '.github' directory tree.
+
+        * 'actionlint' erroneously generates false positives when dealing with GitHub's `${{ ... }}` syntax in conditionals.
+
+        * 'actionlint' erroneously generates incorrect solutions when suggesting the removal of valid `${{ ... }}` syntax.
+    - path: docs/*
+      instructions: >-
+        When a project contains a `docs/` directory:
+
+        1. Consider the files in the `docs/` directory tree the core/main/in-depth documentation of the project. Also consider the 'docs/**.md' files the second place to look for project documentation after the 'README.md' file.
+
+        2. When reviewing the documentation files (e.g. `docs/**.md`), they should additionally be linted with help from the tool `markdownlint`, pointing out any issues.
+
+        3. When reviewing the documentation files in `docs/` directory, they should additionally be linted with help from the tool `languagetool`, pointing out any issues.
+  abort_on_close: true
+  auto_review:
+    enabled: true
+    auto_incremental_review: true
+    ignore_title_keywords: []
+    labels: []
+    drafts: false
+    base_branches:
+      - stable
+      - master
+      - HOTFIX-*
+  tools:
+    languagetool:
+      enabled: true
+      language: en-US
+      configuration:
+        level: picky
+        mother_tongue: en
+        dictionary:
+          - 'reactive-firewall'
+          - 'CEP-9'
+          - 'CEP-8'
+          - 'CEP-7'
+          - 'CEP-5'
+          - 'CEP-4'
+          - 'Shellscript'
+          - 'bash'
+        disabled_rules:
+          - EN_QUOTES
+          - CONSECUTIVE_SPACES
+        enabled_rules:
+          - STYLE
+          - EN_CONTRACTION_SPELLING
+          - EN_WORD_COHERENCY
+          - IT_IS_OBVIOUS
+          - TWELFTH_OF_NEVER
+          - OXFORD_SPELLING
+          - PASSIVE_VOICE
+    shellcheck:
+      enabled: true
+    ruff:
+      enabled: true
+      configuration:
+        extend_select:
+          - E  # Pycodestyle errors (style issues)
+          - F  # PyFlakes codes (logical errors)
+          - W  # Pycodestyle warnings
+          - N  # PEP 8 naming conventions
+        ignore:
+          - W191
+          - W391
+          - E117
+          - D208
+        line_length: 100
```

Comment:
Ignoring documentation rules but not ensuring documentation rules are selected (see above))